common/djangoapps/util/help_context_processor.py

```diff
@@ -67,10 +67,11 @@ def get_doc_url():
             # as the base of documentation link URLs. If it is not set, the
             # function reads the base of the documentation link URLs from
             # the .ini configuration file, lms_config.ini or cms_config.ini.
-            if settings.DOC_LINK_BASE_URL:
-                doc_base_url = settings.DOC_LINK_BASE_URL
-            else:
-                doc_base_url = config_file_object.get("help_settings", "url_base")
+            if doc_base_url is None:
+                if settings.DOC_LINK_BASE_URL:
+                    doc_base_url = settings.DOC_LINK_BASE_URL
+                else:
+                    doc_base_url = config_file_object.get("help_settings", "url_base")
```

Comment:
What's the deal with this line? Looks like we're squashing and rebasing, but I'd assumed anything that needed removing would be removed beforehand.